.ci_support/build_all.py

```diff
@@ -207,9 +240,47 @@ def build_folders(recipes_dir, folders, arch, channel_urls):
         conda_build.api.build([recipe], config=get_config(arch, channel_urls))
 
 
+def build_folders_rattler_build(recipes_dir: str, platform, arch, channel_urls: list[str]):
+    config = get_config(arch, channel_urls)
+
+    # Remove the example recipes to ensure that they are not also build.
+    for example_recipe in EXAMPLE_RECIPE_FOLDERS:
+        rmtree(os.path.join(recipes_dir, example_recipe), ignore_errors=True)
+    
+    # Determine the locations for the variant config files.
+    specs = OrderedDict()
+    for f in config.exclusive_config_files:
+        specs[f] = conda_build.variants.parse_config_file(os.path.abspath(os.path.expanduser(os.path.expandvars(f))), config)
```

Comment:
Does this get the `recipes/xxx/conda_build_config.yaml` too?

Reply:
We just implemented support for a `variants.yaml` file that is adjacent to the `recipe.yaml` (xref: https://github.com/prefix-dev/rattler-build/pull/980)

Reply:
We just implemented support for a `variants.yaml` next to the `recipe.yaml`: https://github.com/prefix-dev/rattler-build/pull/980